ls1intum/Artemis
# `Tutorial groups`: Fix a usability issue for creating new tutorial for campus and language suggestions

### Checklist
#### General


#### Server
#### Client

### Motivation and Context


This PR is designed to improve usability when creating tutor groups. The PR offers the default selection of English and German. For the campus search, the default values have been extended. I.e. all possible values of the campus the instructor is involved in are now offered - not only the possibilities of the respective course

### Description

fixes https://github.com/ls1intum/Artemis/issues/7292

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Courses where the user is instructor
- 2 Tutor Groups

1. Log in to Artemis
2. Navigate to Course Administration
3. Create one Tutorial --> check whether the default values for english and german are available.  
4. Switch to 2nd course --> create another Tutorial 
5. Check if the previouly inserted campus is visible. 
6. Repeat steps for editing of a tutorial


### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage






| Class/File | Line Coverage | Confirmation (assert/expect) |
|------------|--------------:|-----------------------------:|
| create-tutorial-group.component.ts | 98% | ✅              |

### Screenshots
![image](https://github.com/ls1intum/Artemis/assets/80524502/650d8331-c7a8-4205-9a5c-6441410b8755)
![image](https://github.com/ls1intum/Artemis/assets/80524502/3da1b4a1-4b12-4b46-a567-e2e1688c59f1)
